config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: aws/amazon-cloudwatch-agent-operator
-  newTag: latest
+  newName: aws/cloudwatch-agent-operator
+  newTag: 1.0.2
```

Comment:
This was made to be consistent with the image produced in the Makefile by `make container`